modules/ROOT/pages/7.1.1-release-notes.adoc

```diff
@@ -168,6 +168,15 @@ As a result, emoji's now render in all occasion without redirecting.
 
 For more information about these changes, see xref:emoticons.adoc#emoticons_images_url[Emoticons#emoticons_images_url].
 
+==== Iframes in dialogs were not rendering rounded borders correctly.
+// #TINY-10901
+
+Previously, when opening a dialog with an `iframe` element that had a different background color than the dialog, the corners of the dialog did not round off as expected as the border radius was applied to the dialog and not the iframe.
```

Comment:
```suggestion
Previously, when opening a dialog with an `iframe` element that had a different background color than the dialog, the corners of the dialog did not round off as expected, as the border radius was applied to the dialog and not the iframe.
```

Reply:
Maybe change `radius was applied to the dialog and not the iframe.` to `radius was applied to the iframe parent element and not the iframe.`

Reply:
```suggestion
Previously, when opening a dialog with an `iframe` element that had a different background color than the dialog, the corners of the dialog did not round off as expected as the border radius was applied to the `iframe` parent element and not the `iframe`.
```